### PR TITLE
Compatibility to Astra Header/Footer Builder.

### PR DESCRIPTION
**Pull request by SuryawanshiPrajakta**
### Description
Added compatibility to the new Astra Header/Footer builder by removing the Astra HF builder actions.

### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?
- With new HF builder by creating the header, footer, and before footer in EHF
- If the new Header/Footer builder is disabled 
- With old Astra theme

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by vrundakansara — APPROVED** [review]

**Review by vrundakansara — APPROVED** [review]
Looks good to me. We can merge it. 🎉

File: inc/class-header-footer-elementor.php [hunk 1]
```diff
@@ -61,6 +61,65 @@ function __construct() {
...
+								array_push(
+									$hf_builder_notices,
+									[
+										'name'     => ASTRA_THEME_SETTINGS . '[footer-notice]',
+										'type'     => 'control',
+										'control'  => 'ast-description',
+										'section'  => 'section-footer-builder-layout',
+										'priority' => 9,
+										'help'     => '<div class="ast-customizer-notice wp-ui-highlight"><p>' . $message . '<p></div>',
+										'context'  => Astra_Builder_Helper::$general_tab,
```
Comment (vrundakansara) [inline inc/class-header-footer-elementor.php:108]: Let's remove this `context`

File: inc/class-header-footer-elementor.php [hunk 2]
```diff
@@ -61,6 +61,65 @@ function __construct() {
...
+								array_push(
+									$hf_builder_notices,
+									[
+										'name'     => ASTRA_THEME_SETTINGS . '[header-notice]',
+										'type'     => 'control',
+										'control'  => 'ast-description',
+										'section'  => 'section-header-builder-layout',
+										'priority' => 9,
+										'help'     => '<div class="ast-customizer-notice wp-ui-highlight"><p>' . $message . '<p></div>',
+										'context'  => Astra_Builder_Helper::$general_tab,
```
Comment (vrundakansara) [inline inc/class-header-footer-elementor.php:86]: Let's remove this `context`

Comment (SuryawanshiPrajakta) [reply inc/class-header-footer-elementor.php]: Done!

File: inc/class-header-footer-elementor.php (hunk 1 above)
Comment (SuryawanshiPrajakta) [reply inc/class-header-footer-elementor.php]: Done!

File: inc/class-header-footer-elementor.php [hunk 3]
```diff
@@ -61,6 +61,63 @@ function __construct() {
...
+										'name'     => ASTRA_THEME_SETTINGS . '[header-notice]',
+										'type'     => 'control',
+										'control'  => 'ast-description',
+										'section'  => 'section-header-builder-layout',
+										'priority' => 9,
+										'help'     => '<div class="ast-customizer-notice wp-ui-highlight"><p>' . $message . '<p></div>',
+									]
+								);
+
+								$hf_builder_notices = array_merge( $hf_builder_notices, Astra_Builder_Base_Configuration::prepare_advanced_tab( 'section-header-builder-layout' ) );
```
Comment (vrundakansara) [inline inc/class-header-footer-elementor.php:89]: Do we need this?
```php
Astra_Builder_Base_Configuration::prepare_advanced_tab( 'section-header-builder-layout' 
```

File: inc/class-header-footer-elementor.php [hunk 4]
```diff
@@ -61,6 +61,63 @@ function __construct() {
...
+								$message = sprintf( __( 'The header on the page you are previewing is built using %3$s plugin. Options given below will not work here.</p><p></br> %1$s Click here %2$s to modify the header on this page.', 'astra' ), '<a href="' . admin_url( 'edit.php?post_type=elementor-hf' ) . '" target="_blank">', '</a>', '<strong>Elementor - Header, Footer & Blocks</strong>' );
+
+								array_push(
+									$hf_builder_notices,
+									[
+										'name'     => ASTRA_THEME_SETTINGS . '[header-notice]',
+										'type'     => 'control',
+										'control'  => 'ast-description',
+										'section'  => 'section-header-builder-layout',
+										'priority' => 9,
```
Comment (vrundakansara) [inline inc/class-header-footer-elementor.php:84]: Set the priority as 1

File: inc/class-header-footer-elementor.php [hunk 5]
```diff
@@ -61,6 +61,63 @@ function __construct() {
...
+							$hf_builder_notices = [];
+
+							if ( hfe_header_enabled() ) {
+								/* translators: %s: Links to to plugin */
+								$message = sprintf( __( 'The header on the page you are previewing is built using %3$s plugin. Options given below will not work here.</p><p></br> %1$s Click here %2$s to modify the header on this page.', 'astra' ), '<a href="' . admin_url( 'edit.php?post_type=elementor-hf' ) . '" target="_blank">', '</a>', '<strong>Elementor - Header, Footer & Blocks</strong>' );
+
+								array_push(
+									$hf_builder_notices,
+									[
+										'name'     => ASTRA_THEME_SETTINGS . '[header-notice]',
```
Comment (vrundakansara) [inline inc/class-header-footer-elementor.php:80]: Keep it `[ehf-header-notice]` / `[ehf-footer-notice]`

**Review by vrundakansara — COMMENTED** [review]
@SuryawanshiPrajakta - Please add these noted changes.

**Review by vrundakansara — APPROVED** [review]
Looks good to me. 🎉

**Pull request merged**